services/brig/test/integration/API/User/Client.hs

```diff
@@ -163,7 +163,7 @@ testTooManyClients opts brig = do
 
     -- There is only one temporary client, adding a new one
     -- replaces the previous one.
-    forM_ [0..(9 :: Int)] $ \i ->
+    forM_ [0..(3 :: Int)] $ \i ->
```

Comment:
Not 100% sure about this, since 9 explicitly exceeds the default max permitted clients. So if for some reason temporary clients get treated as permanent, this test would (if I understand correctly) fail with 9, but succeed with 3.

Reply:
good point.  this should be pulled into the block with the changed settings.  or just rolled back.

Reply:
I'll pull it into the settings block, we should be explicit about the configuration values we expect.